### ISSUE TITLE
How to use?

### ISSUE THREAD

**Issue by Kralizek** [tintoy/dotnet-template-templates#3]
Hi!

I was looking through the scarce documentation currently existing about the new templating system and I found your project.
Although I couldn't find any way to use it.
Should I download the source or is there a NuGet package I can consume?

**Comment by tintoy** [comment]
Hi - thanks for reminding me, I need to update the documentation :smile:

To get going for  now, all you need to do is fetch the source and, from the root of the repository, run:

```
dotnet new -i src/Templates/content
```

I'll update the README shortly, and publish a NuGet package early next week.

**Comment by Kralizek** [comment]
Thank you! :)

**Comment by Kralizek** [comment]
For some reason, `dotnet new -i src\Templates\content` seems like ignored. 

![image](https://cloud.githubusercontent.com/assets/2458439/24084981/4cebde7c-0cf4-11e7-892a-3eaece7dcd1d.png)

**Comment by tintoy** [comment]
Sorry!  I didn't receive a notification when you reopened this. I'm about to reorganise the templates so I'll test this out for you shortly. Sorry for the inconvenience :-/

**Comment by tintoy** [comment]
By the way, does it work for you when you install from a package, @Kralizek?

**Comment by tintoy** [comment]
Ok, I tested this using the 1.0.4 tooling, and it seems to work correctly.

![image](https://user-images.githubusercontent.com/1646284/27990243-9bd9d808-6493-11e7-8895-ab78b9899f1e.png)

Are you able to run `dotnet --version` and post the output here?

**Comment by doggy8088** [comment]
@tintoy The `dotnet new template -t project` is not working in the latest build that running in my .NET Core 3.1.300 SDK.

**Comment by tintoy** [comment]
Hmm - I haven’t really kept track of the template engine changes for a while, I’ll take a look this weekend and see what sort of changes are required :)

**Comment by tintoy** [comment]
@doggy8088 - what behaviour are you seeing when you try to run `dotnet new template -t project`?

**Comment by tintoy** [comment]
@doggy8088 can you try this one?

[FiftyProtons.Templates.DotNetNew.3.1.0-alpha.1.zip](https://github.com/tintoy/dotnet-template-templates/files/4672756/FiftyProtons.Templates.DotNetNew.3.1.0-alpha.1.zip)

**Comment by doggy8088** [comment]
Is this the right usage?

![image](https://user-images.githubusercontent.com/88981/82761630-8bd68f00-9e2e-11ea-8139-4158da0cce10.png)

**Comment by doggy8088** [comment]
> @doggy8088 - what behaviour are you seeing when you try to run dotnet new template -t project?

I'd like to transform my existing project into a template.

**Comment by tintoy** [comment]
Try unzipping the package above and run `dotnet new -i xxx.nukg` first?

**Comment by doggy8088** [comment]
I did.

**Comment by doggy8088** [comment]
It looks like there is no `dotnet new template` command.

![image](https://user-images.githubusercontent.com/88981/82762166-12d93680-9e32-11ea-9b39-cc75defbb927.png)

**Comment by tintoy** [comment]
Oh I understand now. you want `dotnet new -t project-template` or `dotnet new -t packaging-template`.

**Comment by doggy8088** [comment]
@tintoy  You should probably edit your README file in your [homepage](https://github.com/tintoy/dotnet-template-templates) of your repo.

![image](https://user-images.githubusercontent.com/88981/82849749-e275c300-9f2b-11ea-99ae-2f6d34cdadb9.png)

**Comment by doggy8088** [comment]
What if I want to convert my existing project into a project template, is there any easy way to do that?

**Comment by tintoy** [comment]
Thanks! Updated 🙂 

> What if I want to convert my existing project into a project template, is there any easy way to do that?

From memory if you have the VS SDK installed, it adds a menu item to the File menu to create a template from the current project. I don't know of any other automated way to do it (I usually do them by hand myself because I do it so infrequently 😄).

**Comment by doggy8088** [comment]
No, what I asked is the way to make my existing .NET Core project into a `dotnet new` template by using your tool.

**Comment by tintoy** [comment]
Ah. I thought they had unified the 2 template mechanisms. If not then no sorry you’ll need to do it by hand.

**Comment by doggy8088** [comment]
I think there is no `-t` option at this version.

![image](https://user-images.githubusercontent.com/88981/82870584-ca6d6600-9f62-11ea-9179-b45ccdc41e0d.png)

**Comment by tintoy** [comment]
Ugh, I wish they would just pick one and stick with it...

Thanks I’ll update it tomorrow morning :)

**Comment by doggy8088** [comment]
@tintoy I updated for you. See PR https://github.com/tintoy/dotnet-template-templates/pull/9
